src/app/parcels/form/ParcelForm.tsx

```diff
@@ -92,13 +99,12 @@ interface ParcelFormProps {
     initialFields: ParcelFields;
     initialFormErrors: ParcelErrors;
     clientId?: string;
-    editMode: boolean;
-    parcelId?: string;
     deliveryPrimaryKey: Schema["collection_centres"]["primary_key"];
     collectionCentresLabelsAndValues: CollectionCentresLabelsAndValues;
     packingSlotsLabelsAndValues: PackingSlotsLabelsAndValues;
-    packingSlotIsShown?: boolean;
-    collectionCentreIsShown?: boolean;
+    writeParcelInfoToDatabase:
+        | ((parcelRecord: ParcelDatabaseUpdateRecord) => Promise<UpdateParcelReturnType>)
+        | ((parcelRecord: ParcelDatabaseInsertRecord) => Promise<InsertParcelReturnType>);
```

Comment:
I think we should define these types in clientDatabaseFunctions.ts, see my comment there